### DISCUSSION TITLE
RGB rainbow fade effect using H801

### DISCUSSION THREAD

**Discussion by julred** [arendst/Tasmota#13615] [Q&A] [answered]
Dear all,

I have a H801 RGBCCT controller. I'd like to visualize a rainbow color fade. (slowly changing from red, to yellow, green, blue -> red again...). Is it possible to run this kind of fade effect directly on tasmota or do I need to calculate the rgb settings in my openhab instance and sending all values each 0,5 sec via mqtt?

**Comment by ascillato** [comment, marked as answer]
Yes, please check the docs at https://tasmota.github.io/docs/Commands/#scheme
